### ISSUE TITLE
Reg\Facility - Internal User can view Single Facility Information page in read only mode

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#2378]
#### Description:
As an internal user, I can see a Single Facility Operation's Facility in read only mode

#### Acceptance Criteria:
Given I am on the [operations grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7861-688&t=82xbIhfIx0CoJNIH-4)
If the operation type is SFO
Then the button under Facilities says "View Facility" (same as external users button)

Given that I click on "View Facility", 
Then it will redirect to the Facility Information page (instead of the facilities grid that is shown for an LFO)
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=18140-1025&t=0E4HRE1djEXL07dW-4)

The Issue BCGHGID buttons on the page are being handled in ticket #2152 

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies
#1701 

- Blocked by
- Blocking

**Comment by patrickisaac** [comment]
Closing ticket - work being captured in #1704
